docs/reference/concepts/02-provider.mdx

```diff
@@ -246,3 +248,63 @@ func (e MyFeatureProvider) Hooks() []Hook {
 </TabItem>
 
 </Tabs>
+
+### Checklist
+
+Here is a checklist of important considerations when implementing a provider.
+
+#### Functional requirements
+
+<CopyChecklist items={[
+  'All resolution methods are implemented (see `Provider` / `FeatureProvider` interface in applicable SDK).',
+  'Relevant configuration options for the underlying backend are exposed (typically using the provider constructor).',
+  'Flag-keys are transformed appropriately for the underlying backend (OpenFeature defines no restrictions in valid flag keys).',
+  'Context is transformed appropriately for the underlying backend.',
+  'Provider is named according to the [naming conventions](#naming_recommendations)'
+]} />
+
+#### Non-functional requirements
+
+<CopyChecklist items={[
+  'Documentation for installation, all behavior (options, context transformation, flag-key transformation).',
+  'Provider is adequately tested.',
+  'Provider is added to [the documentation site](https://github.com/open-feature/docs.openfeature.dev/issues/new?assignees=&labels=hook&template=document-hook.yaml&title=%5BHook%5D%3A+).',
+  'Provider releases follow [semantic versioning](https://semver.org/).',
+  'Document the version of the OpenFeature specification to which your provider complies. '
+]} />
+
+### FAQ
+
+#### Does my provider's version need to match the spec version to which it complies?
+
+No. You should document the version of the OpenFeature specification your provider complies to, and otherwise use [semantic versioning](https://semver.org/) in your artifact numbers.
+
+#### My backend need a targeting key (or other user identifier), how can I ensure one exists?
+
+Targeting keys are not required by the Evaluation API. If your backend requires one, you have a couple options:
+
+- Indicate an error in your provider (this will result in the default value being returned).
```

Comment:
Perhaps add a bullet point item to add the requirement to the provider's readme.

Reply:
Added "Be sure to document this behavior."